app/src/main/java/io/intrepid/webservicesexercise/MainActivity.java

```diff
@@ -28,29 +38,44 @@ protected void onCreate(Bundle savedInstanceState) {
         ButterKnife.bind(this);
 
         Timber.plant(new Timber.DebugTree());
+
     }
 
+    private static GitHubApi createRestApi() {
+        OkHttpClient.Builder builder = new OkHttpClient().newBuilder();
+        OkHttpClient httpClient = builder
+                .connectTimeout(2, TimeUnit.SECONDS)
+                .build();
+        return new Retrofit.Builder()
+                .baseUrl(GITHUB_BASE_URL)
+                .client(httpClient)
+                .addConverterFactory(GsonConverterFactory.create())
+                .build()
+                .create(GitHubApi.class);
+    }
     @Override
     protected void onDestroy() {
```

Comment:
now that there's no additional logic in this method, the whole thing can be deleted